dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -114,6 +114,36 @@ Configuration InstallCAM
 		}
 
 
+        File Sumo_Directory 
+        {
+            Ensure          = "Present"
+            Type            = "Directory"
+            DestinationPath = "C:\sumo
+        }
+        # Aim to install the collector first and start the log collection before any 
+        # other applications are installed.
+        Script Install_SumoCollector
+        {
+            DependsOn  = "[File]Sumo_Directory"
+            GetScript  = { @{ Result = "Install_SumoCollector" } }
+
+            SetScript  = {
+                Write-Verbose "Install_SumoCollector"
+
+                $sumo_package = "https://teradeploy.blob.core.windows.net/binaries/SumoCollector-windows-x64_19_182-25.exe"
+                $sumo_config = "/new-admin-vm/sumo.conf"
+                $sumo_collector_json = "/new-admin-vm/sumo-admin-vm.json"
+                $dest = "C:\sumo"
+                Invoke-WebRequest $sumo_config -OutFile "$dest\sumo.conf"
```

Comment:
You can use xRemoteFile's for getting source and binaries. I'd recommend that since they do some retries, etc. automatically.